src/main/java/org/prebid/server/bidder/adview/AdviewBidder.java

```diff
@@ -97,61 +87,45 @@ private ExtImpAdview parseExtImp(Imp imp) {
         }
     }
 
-    private static boolean shouldConvertBidFloor(Imp imp) {
-        return BidderUtil.isValidPrice(imp.getBidfloor())
-                && !StringUtils.equalsIgnoreCase(imp.getBidfloorcur(), BIDDER_CURRENCY);
-    }
-
-    private BigDecimal resolveBidFloor(BidRequest request, Imp imp, String bidFloorCurrency) {
-        final BigDecimal bidFloor = imp.getBidfloor();
-        final String bidFloorCur = imp.getBidfloorcur();
-
-        return !bidFloorCurrency.equals(bidFloorCur)
-                ? convertBidFloor(bidFloor, bidFloorCur, imp.getId(), request)
-                : bidFloor;
+    private Price resolveBidFloor(Imp imp, BidRequest bidRequest) {
+        final Price initialPrice = Price.of(imp.getBidfloorcur(), imp.getBidfloor());
```

Comment:
Minor. In method parameters you used `floorPrice` naming. So, for here `initialFloorPrice`

Reply:
maybe even stick to `initialBidFloorPrice`